krun.py

```diff
@@ -140,13 +125,16 @@ class ScheduleEmpty(Exception):
 class ExecutionScheduler(object):
     """Represents our entire benchmarking session"""
 
-    def __init__(self, config_file, log_filename, out_file, mailer, platform):
+    def __init__(self, config_file, log_filename, out_file, mailer, platform,
```

Comment:
I hesitate to call it that as we already have a dict called `config' floating about. What do you think?